packages/storybook/stories/va-text-input-uswds.stories.jsx

```diff
@@ -392,7 +384,27 @@ WithCharacterCount.args = { ...defaultArgs, maxlength: '10', charcount: true };
 export const WithCurrency = Template.bind(null);
 WithCurrency.args = {
   ...defaultArgs,
-  currency: true
+  currency: true,
+};
+
+export const WithCreditcard = Template.bind(null);
```

Comment:
Instead of naming this story "With Credit Card", can we make it more general like "With Icon"?

![Screenshot 2024-07-03 at 10 22 46 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/b9c98c6c-127c-4a3b-b973-c6edadd48f24)